dbt_copilot_helper/templates/addons/env/aurora-postgres.yml

```diff
@@ -151,3 +152,309 @@ Resources:
       FromPort: 5432
       ToPort: 5432
       SourceSecurityGroupId: !Ref EnvironmentSecurityGroup
+
+  # Enable ingress from Lambda Functions.
+  {{ service.prefix }}LambdaIngress:
+    Metadata:
+      'aws:copilot:description': 'Allow ingress from Lambda functions in my application to the DB cluster'
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      Description: Ingress from Lambda Function
+      GroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+      IpProtocol: tcp
+      FromPort: 5432
+      ToPort: 5432
+      SourceSecurityGroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+
+  # Enable ingress from Secrets Manager.
+  {{ service.prefix }}SecretsManagerIngress:
+    Metadata:
+      'aws:copilot:description': 'Allow ingress from Secrets Manager in my application to the DB cluster'
```

Comment:
Why does Secrets Manager need ingress to the database cluster? Maybe we can add a little more detail into some of the decsriptions?

Reply:
It doesn't, and the description is wrong. One thing to note though is that I have placed the Lambda Function in the same security group as the database cluster.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/185/commits/a385eceb3ab4c605beb82f257a738a8b87386a51.